pages/linux/userdel.md

```diff
@@ -1,7 +1,17 @@
 # userdel
 
-> Remove a user.
+> Remove a user account or remove a user from a group.
+> Note: all commands must be executed as root.
+> More information: <https://man7.org/linux/man-pages/man8/userdel.8.html>.
 
-- Remove a user and their home directory:
+- Remove a user:
+
+`userdel {{name}}`
+
+- Remove a user along with their home directory and mail spool:
 
 `userdel -r {{name}}`
+
+- Remove a user from a group:
+
+`userdel {{name}} {{group}}`
```

Comment:
We can add an example for `--root`:
```suggestion
`userdel {{name}} {{group}}`

- Remove a user in a new root directory:

`userdel --root {{path/to/new/root}} {{name}}`
```

Reply:
@navarroaxel that description is a little unclear...could you please improve it? (only if possible, because the man page isn't clear either)

Reply:
Or clarify me if I'm just misreading it :smile:

Reply:
I tried to use a similar description from the `chroot` page.
🤔 imagine you're in a live USB and you want to remove a user from your installed Linux. So mount `/dev/sda1` into `/mnt`, then run `userdel --root /mnt user1`. This commands in like `chroot /mnt` and  `userdel user1` combined.

Reply:
Ah I see!  Very good explanation 👍🏻  Now how would you get that succinctly into the exmaple description?

Reply:
Added this, but changed "new" to "other"